modules/docs/mdx/6.0-MIGRATION-GUIDE.mdx

```diff
@@ -364,3 +371,47 @@ will have two effects for this component:
   - This was previously happening only on viewports < 576px wide
 
 These changes in behavior are intentional and expected.
+
+## Button Updates
+
+`PrimaryButton`, `SecondaryButton`, and `TertiaryButton` have been slightly adjusted to tighten up
+the spacing and use our new type hierarchy. We also adjusted the opacity for their disabled state
+for a bit more consistency. These changes are automatic when you upgrade and do not require any
+manual updates in your codebase. However, these changes will likely cause any automated visual
+regression tests to fail, and you will need to update them accordingly. These adjustments are fairly
+minor, and we do not expect them to create significant layout shifts in your UI. But, as with any
+visual update, you will want to review your UI to ensure there are no adverse effects. We also added
+some net-new styling: `PrimaryButton`, `SecondaryButton`, and `TertiaryButton` have a new
+extra-small size, and `PrimaryButton` has a new `inverse` variant.
```

Comment:
I would try and break out this text into more digestable info, there's a lot of important stuff here.

`PrimaryButton`, `SecondaryButton`, and `TertiaryButton` have been slightly adjusted to tighten up
the spacing and use our new type hierarchy. 

- We also adjusted the opacity for their disabled statefor a bit more consistency. These changes are automatic when you upgrade and do not require any. (*Note* these changes will likely cause any automated visual
regression tests to fail, and you will need to update them accordingly.)

- We also added some net-new styling: `PrimaryButton`, `SecondaryButton`, and `TertiaryButton` have a new
extra-small size

- `PrimaryButton` has a new `inverse` variant.